Enterprise-CMCS/eAPD
# 4575 State Priorities and Scope admin check issue

Resolves #4575 

### Description
Resolves bug where the validation was triggering in State Priorities fields even when the admin check was not enabled

### Steps to manually verify this change

1. Login to MMIS apd
2. Verify that the inline validation does not trigger when the admin check is turned off

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when QA has


### This pull request can be merged when
